### PR TITLE
testing/efl: new aport

### PR DESCRIPTION
**Pull request by billiob**

### CODE REVIEW

File: testing/efl/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Contributor: Boris Faure <boris@fau.re>
+# Maintainer:
```
Comment (Ikke) [inline testing/efl/APKBUILD]: The package should have a maintainer. Do you want to maintain this package?

File: testing/efl/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+build() {
+	cd "$builddir"
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--localstatedir=/var \
+                --enable-image-loader-png \
```
Comment (Ikke) [inline testing/efl/APKBUILD]: The indentation is inconsistent here and a few other lines.

Comment (billiob) [reply testing/efl/APKBUILD]: I should have fixed those.

File: testing/efl/APKBUILD (hunk 1 above)
Comment (billiob) [reply testing/efl/APKBUILD]: Yes, I do.

**Comment by Ikke** [comment]
From the build log:

> \>>> WARNING: efl-dev*: Could not find any provider for pc:luajit>=2.0.0                                                                                                                                                                        
>>> WARNING: efl-dev*: luajit-dev needs to be rebuilt                                                                                                                                                                                          
>>> WARNING: efl-dev*: openssl-dev should be removed from depends_dev

**Comment by billiob** [comment]
Do you have any pointers on how I could fix the first two warnings?

**Comment by billiob** [comment]
I've opened https://bugs.alpinelinux.org/issues/10089

**Comment by billiob** [comment]
I've fixed the last issue.

**Comment by clandmeter** [comment]
@billiob can you rebase against master? it will enable drone CI.

**Comment by billiob** [comment]
I've just done it again.

**Comment by Ikke** [comment]
It still gives warnings about luajit, no idea if that is an issue or not.

**Comment by billiob** [comment]
According to the [bug report](https://bugs.alpinelinux.org/issues/10089), the issue is in `apk` itself.
The packages built are working in my case to build and test [Terminology](https://github.com/billiob/terminology).

**Comment by Ikke** [comment]
Sadly, the test suite does not seem to run any tests:

> ============================================================================
Testsuite summary for efl 1.21.1
\============================================================================
\# TOTAL: 0
\# PASS:  0
\# SKIP:  0
\# XFAIL: 0
\# FAIL:  0
\# XPASS: 0
\# ERROR: 0
\============================================================================

In that case, you should disable the test suite (provide `options="!check" # no tests provided`

Also, please wrap the list of dependencies around 80 characters.

**Comment by billiob** [comment]
There shall be tests. I'll try to figure that out.

**Comment by billiob** [comment]
I got rid of the checks since they are failing on this version.

**Pull request closed without merging**